lib/event_listeners.js

```diff
@@ -80,7 +80,7 @@ AWS.EventListeners = {
       req.service.config.getCredentials(function(err) {
         if (err) {
           req.response.error = AWS.util.error(err,
-            {code: 'CredentialsError', message: 'Missing credentials in config'});
+            {code: 'CredentialsError', message: 'Missing credentials in config, try to run \'AWS_SDK_LOAD_CONFIG=1 npm start\''});
```

Comment:
@AllanFly120 

How about this message instead:

> Missing credentials in config, if using AWS_CONFIG_FILE, set AWS_SDK_LOAD_CONFIG=1

Reply:
@srchase sure!